fiboa_cli/datasets/commons/euro_land.py

```diff
@@ -0,0 +1,50 @@
+class EuroLandConverterMixin:
+    """
+    Datasets have been published by the
+    [Euroland project](https://europe-land.eu/news/harmonized-database-of-european-land-use-data-published/)
+    as open data. See https://zenodo.org/records/14384070 for a list of open data sets.
+
+    Use this abstract class to create converters based on the euroland repository
+    Subclasses should declare: id, short_name, title, description, providers, license, attribution,
+    crop_code_list,
+    """
+    crop_code_list = None  # e.g. ec_url(ec_mapping_csv)
+    extensions = [
+        "https://fiboa.github.io/crop-extension/v0.1.0/schema.yaml",
+        "https://fiboa.github.io/hcat-extension/v0.1.0/schema.yaml"
+    ]
+    columns = {
+        "geometry": "geometry",
+        "field_id": "id",
+        "farm_id": "farm_id",
+        "crop:code_list": "crop:code_list",
+        "crop_code": "crop:code",
+        "crop_name": "crop:name",
+        'EC_trans_n': 'ec:translated_name',
+        'EC_hcat_n': 'ec:hcat_name',
+        'EC_hcat_c': 'ec:hcat_code',
+        "organic": "organic",
```

Comment:
I  haven't had a look at the dataset itself and didn't find a data survey, so I'm wondering what organic may contain? From the field name it sounds like a boolean reather than a string.

Reply:
Fair enough. Should the schema then be uint8 instead of string?